src/data-source.ts

```diff
@@ -0,0 +1,19 @@
+import 'reflect-metadata'
+import { DataSource } from 'typeorm'
+import { User } from './entity/User'
+import { Team } from './entity/Team'
+import { WorkPeriod } from './entity/WorkPeriod'
+
+export const AppDataSource = new DataSource({
+    type: 'postgres',
+    host: 'localhost',
+    port: 5432,
+    username: 'postgres',
+    password: '',
+    database: 'timetracking',
+    synchronize: true,
+    logging: true,
+    entities: [User, Team, WorkPeriod],
```

Comment:
Instead of manually listing ﻿[User, Team, WorkPeriod], consider using a wildcard import (﻿../entities/*.entity.ts) for entities in the specific folder